archai/nlp/nvidia_transformer_xl/nvidia_utils/proj_adaptive_softmax.py

```diff
@@ -183,7 +191,7 @@ def clean_tie_projs(tie_projs:Optional[List[bool]], cutoffs:List[int], n_token:i
         return tie_projs[:len(cutoffs)]
 
     def get_out_proj(self, i):
-        if self.tie_projs[i]:
+        if not self.tie_projs or self.tie_projs[i]:
```

Comment:
I thought this was handled. If tie_projs is empty it would be reset to proper value.